Header header
string sensor_id
string date
string status
string trigger
string triggerName
